src/ILink.php

```diff
@@ -47,4 +47,9 @@ public function getRole() : string;
 	 * @return string
 	 */
 	public function getRel() : string;
+
+	/**
+	 * @return string
+	 */
+	public function getTarget() : string;
```

Comment:
This is a public interface change and requires a major version release. Maybe just create a new interface `ITargetArware`?